File: Proyecto/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Proyecto
{
  public partial class Form2 : Form
  {
    public Form2()
    {
      InitializeComponent();
    }

    public string Vote { get; set; }

    public string NewVote { get; set; }

    private void button1_Click(object sender, EventArgs e)
    {
      Form3 form3 = new Form3();
      form3.Vote = "NewVote";
      form3.Show();

    }

    private void button2_Click(object sender, EventArgs e)
    {
      Form3 form3 = new Form3();
      form3.Vote = "NewVote";
      form3.Show();
    }

    private void button1_Click_1(object sender, EventArgs e)
    {

    }
  }
}

File: Proyecto VB/Form2.vb
Imports System.IO
Imports System.Reflection.Metadata
Imports System.Windows.Forms.VisualStyles.VisualStyleElement.Tab


Public Class Form2
    Private votante As Votante
    Public Sub New()
        InitializeComponent()
        votante = New Votante()
    End Sub

    Private information As String() = New String(8) {}

    Public Property Vote As String
        Get
            Return m_Vote
        End Get
        Set
            m_Vote = Value
        End Set
    End Property
    Private m_Vote As String

    Public Property NewVote As String
        Get
            Return m_NewVote
        End Get
        Set
            m_NewVote = Value
        End Set
    End Property
    Private m_NewVote As String

    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        Dim form3 As New Form3()
        form3.Vote = "NewVote"
        form3.Show()
        SaveWord()
    End Sub

    Private Sub button2_Click(sender As Object, e As EventArgs) Handles button2.Click
        Dim form3 As New Form3()
        form3.Vote = "NewVote"
        form3.Show()
    End Sub

    Private Sub SaveWord()
        Dim saveFileDialog As New SaveFileDialog()
        saveFileDialog.Filter = "Documentos de Word (.docx)|*.docx"

        If saveFileDialog.ShowDialog() = DialogResult.OK Then
            Dim filepath As String = saveFileDialog.FileName

            Try
                ' Define namespace aliases
                Dim W As XNamespace = "http://schemas.openxmlformats.org/wordprocessingml/2006/main"

                ' Create a new Word document
                Dim doc As XDocument = New XDocument(
                    New XDeclaration("1.0", "UTF-8", "yes"),
                    New XElement(W + "document",
                        New XElement(W + "body",
                            From info In information
                            Select New XElement(W + "p",
                                New XElement(W + "r",
                                    New XElement(W + "t", info)
                                )
                            )
                        )
                    )
                )

                ' Save the document
                Using stream As New FileStream(filepath, FileMode.Create)
                    doc.Save(stream)
                End Using

                MessageBox.Show("Documento guardado correctamente.", "Guardado", MessageBoxButtons.OK, MessageBoxIcon.Information)
            Catch ex As Exception
                MessageBox.Show("Error al guardar el documento: " & ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Error)
            End Try
        End If
    End Sub
End Class